packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -24,29 +24,6 @@ import {
     KEY__SYNTHETIC_MODE,
     setPrototypeOf,
 } from '@lwc/shared';
-import {
```

Comment:
there are now extracted from the renderer associated to the `vm` for a lightning element.